docs/zkapps/advanced-snarkyjs/events.mdx

```diff
@@ -61,6 +61,34 @@ class SmartContract {
 }
 ```
 
+### Fetching Events from an Archive Node
+
+Since emitted event data is only stored in transaction data, events will only be persisted by an Archive Node. To fetch events from an Archive Node, you can use the `archive` property on the `Mina.Network` function. For example:
+
+```ts
+let Network = Mina.Network({
+  mina: '', // Use https://proxy.berkeley.minaexplorer.com/graphql or https://api.minascan.io/node/berkeley/v1/graphql
+  archive: '', // Use https://api.minascan.io/archive/berkeley/v1/graphql/ or https://archive.berkeley.minaexplorer.com/
+});
+Mina.setActiveInstance(Network);
+```
+
+By setting the `archive` property to one of the listed endpoints, SnarkyJS will use one of the selected endpoints to fetch events. For example, let us assume that a zkApp has already emitted events and has broadcasted the transaction. To fetch events for a zkApp, you can use the `fetchEvents` function:
```

Comment:
Would rephrase b/c the URL isn't the actor. I.e. it's fetched from an endpiont, the endpoint doesn't do it.

`fetchEvents()`